app/util/PasscodeCreator.ts

```diff
@@ -8,7 +9,10 @@ class PasscodeCreator {
   private availableWords: string[];
   private separator: string;
 
-  constructor(filePath = '../../wordList.txt', separator = '-') {
+  constructor(
+    filePath = `../locales/${LANGUAGE}-wordlist.txt`,
```

Comment:
The file is called `nl-wordList`, not `nl-wordlist`

Reply:
Yeah, I had renamed it, but `git` doesn't care about casing. Renamed the files in order to fix the problem (since `nl`and `sk`was different).